packages/compass-connections/src/components/non-genuine-connection-modal.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  css,
+  Banner,
+  Link,
+  spacing,
+  Body,
+  BannerVariant,
+  showConfirmation,
+} from '@mongodb-js/compass-components';
+import {
+  getConnectionTitle,
+  type ConnectionInfo,
+} from '@mongodb-js/connection-info';
+
+const modalBodyStyles = css({
+  marginTop: spacing[3],
+  marginBottom: spacing[2],
+});
+
+export function showNonGenuineMongoDBWarningModal(
+  connectionInfo?: ConnectionInfo
+) {
+  return showConfirmation({
```

Comment:
Button names are slightly different because of this change, but I'm not sure if it matters much tbh, I didn't try to make the names exactly the same